features/SPS004_No-combination-of-containment-and-positioning.feature

```diff
@@ -6,14 +6,14 @@ The rule verifies that each IfcProduct can either be contained in one and only o
 
   Scenario: Agreement on elements being contained cannot be positioned
 
-      Given A file with Schema Identifier "IFC4X3_TC1" or "IFC4X3_ADD1" or "IFC4X3"
+      Given A file with Schema Version "IFC4.3"
```

Comment:
The same question about `Version`. In some places, the keyword is used, in some not.